deploy/draft/template-legacy.yaml

```diff
@@ -0,0 +1,761 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
do we need this in this PR?

Reply:
The entire file? No I was going to delete after checking there's no reason to keep :)